app/Main.hs

```diff
@@ -4,13 +4,11 @@ module Main where
 
 import Control.Concurrent.Async
 import qualified Control.Concurrent.Chan.Unagi as U
-import Control.Monad
 import Data.DateTime (getCurrentTime)
 import qualified Data.Text as Text
 import qualified Data.UUID as UUID
-import System.Environment (getEnv)
+-- import System.Environment (getEnv)
```

Comment:
Remove?

Reply:
This was because I had the other commented out bit of code. It'll get removed once `Main` is less of a hacky-trying-out-the-things dumping ground.